spec/requests/product_drives_requests_spec.rb

```diff
@@ -47,6 +47,23 @@
         expect(response.body).not_to include(product_drive_path(product_drive_five.id))
       end
 
+      it "allows filtering by tag" do
```

Comment:
Should this be a shared example?

Reply:
Do you mean a shared example for filtering product drives? Or a shared example for filtering different resources by tags?

Reply:
Seems premature to me to create a shared example for filtering by tags, since only product drives have tags.

Reply:
But the whole point of the tag feature is to make it a generic way to attach tags to anything - we know we're not going to limit it to product drives.

Reply:
Ok then